examples/Makefile

```diff
@@ -1,5 +1,8 @@
 .PHONY: default
 default:
+	make -C limited-order-book clean
 	make -C limited-order-book
+	make -C chacha20 clean
 	make -C chacha20
+	OTHERFLAGS="--lax" make -C sha256 clean
 	OTHERFLAGS="--lax" make -C sha256
```

Comment:
Ah, I've just added a `clean` target in #344.

Reply:
heh, ok that's cleaner. Undoing this one.